webpack.config.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
   entry: './src/app.js',
   target: 'node',
   node: {
-    __dirname: false,
+    __dirname: true,
```

Comment:
Just curious, what is this for?

Reply:
This is from #263 where the existing false setting made `__dirname` return incorrect results in some cases (I think setting it to false means that `__dirname` is always equal to 'src/'). I just updated #263 to not require this change and instead put its files in the proper location